xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}$.$csv{$query,1}$.$csv{$hitlist{$id$if{$ne{$hit,$sub{$last,1}},$.}},1}$.$topdoc}
```

Comment:
Your example `clicks.log` given in the python script doc comment shows query ids:

```
821f03288846297c2cf43c34766a38f7,54
```

Is that incorrect?

And if it's not logged there, why do we need to pass the query id to the click logging template?